manifests/08-prometheus_rule.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  annotations:
+    include.release.openshift.io/self-managed-high-availability: "true"
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/single-node-developer: "true"
+  name: insights-prometheus-rules
+  namespace: openshift-insights
+spec:
+  groups:
+  - name: insights
+    rules:
+    - alert: InsightsDisabled
+      annotations:
+        description: 'Insights operator is disabled. In order to enable Insights and benefit from recommendations specific to your cluster, please follow steps listed in the documentation: https://docs.openshift.com/container-platform/latest/support/remote_health_monitoring/enabling-remote-health-reporting.html'
+        summary: Insights operator is disabled.
+      expr: |
+       cluster_operator_conditions{name="insights", condition="Disabled"} == 1
```

Comment:
Should we have similar rules/alerts for Insights is 'enabled' but not able to send back information to Red Hat (IE: Running in an isolated network). 

The net effect is the same; "In order to receive benefit from recommendations specific to your cluster" you have to be able to phone home (to Red Hat). 

In the 'how to resolve' docs for this; we may need to talk about proxies; etc.

Reply:
>Should we have similar rules/alerts for Insights is 'enabled' but not able to send back information to Red Hat (IE: Running in an isolated network).


If the Insights operator is not able to upload for some time then it's marked as degraded and yes I think we can have similar alert, but having a degraded cluster operator is a vissible enough problem....at least from my perspective.